### PR TITLE
Mini sabbatical post

### PR DESCRIPTION
**Pull request by zoccoler**
Hi,

I just added Rike's text about her experience last year with us.
I chose an image to decorate it. We may change it if you like it.
We could add some links to her text. What do you guys think?

Best,
Marcelo

### CODE REVIEW

File: marcelo_zoccoler/mini_sabbatical_rike/Readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Mini-Sabbatical Experience at Bia-PoL
+Friederike Kessel, February 28th 2022
+
+Hello, my name is Rike. For the last four years I have been a PhD student in experimental nephrology at the University clinic in Dresden. During this time I started diving into the possibilities of bioimage analysis with Fiji. By using the ImageJ Macro language I helped colleagues to automatize workflows for the quantification of fluorescent signals in histology and intravital microscopy in mice.
```
Comment (haesleinhuepf) [inline marcelo_zoccoler/mini_sabbatical_rike/Readme.md:4]:
```suggestion
Hello, my name is Rike. For the last four years I have been a PhD student in experimental nephrology at the University Hospital Carl Gustav Carus in Dresden. During this time I started diving into the possibilities of bioimage analysis with Fiji. By using the ImageJ Macro language I helped colleagues to automatize workflows for the quantification of fluorescent signals in histology and intravital microscopy in mice.
```

File: marcelo_zoccoler/mini_sabbatical_rike/Readme.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+ 
+In conclusion I can only say: A mini-sabbatical, especially in a non-pressure environment, is the prefect way to gain new perspectives and connect with other scientist. I definitely recommend the general concept itself, and Robert’s group specifically - for anyone, from students to group leaders. I really hope that I will be able to repeat this experience in the future.
```
Comment (haesleinhuepf) [inline marcelo_zoccoler/mini_sabbatical_rike/Readme.md:10]:
```suggestion
In conclusion I can only say: A mini-sabbatical, especially in a non-pressure environment, is the prefect way to gain new perspectives and connect with other scientists. I definitely recommend the general concept itself, and Robert’s group specifically - for anyone, from students to group leaders. I really hope that I will be able to repeat this experience in the future.
```

**Review by haesleinhuepf — APPROVED** [review]
Just two minor suggestions :-)

**Comment by zoccoler** [comment]
Thanks Robert @haesleinhuepf !
Thus, I will merge it.

**Pull request merged**